README.md

```diff
@@ -14,10 +13,22 @@ It will read a configuration file and setup your runs, customize `run.cns` param
 
 ## Installation
 
-
 ```
-$ conda env create -f environment.yml
-$ conda activate benchmark-tools
+$ git clone https://github.com/haddocking/benchmark-tools.git
+$ cd benchmark-tools
+$ python setup.py install
```

Comment:
make it `python setup.py develop`. Since the user will be using the source from github it is best to keep the executable source from the github folder. the `install` will copy the source to `anaconda` (or the python install). with `develop` the changes in the github folder (for example `git pull`) get reflected immediately.